docker_mcp/models/params.py

```
@@ -0,0 +1,195 @@
+        default=None, 
+        description="Docker Compose file path"
+    )
+    enabled: bool = Field(
+        default=True, 
+        description="Whether host is enabled"
+    )
+    ssh_config_path: Optional[str] = Field(
+        default=None, 
+        description="Path to SSH config file"
+    )
+    selected_hosts: Optional[str] = Field(
```

Ruff (0.12.2)

Use `X | None` for type annotations

Convert to `X | None`

(UP045)